Tools/GNUMake/sites/Make.nersc

```diff
@@ -24,6 +24,22 @@ ifeq ($(which_computer),$(filter $(which_computer),perlmutter))
   endif
   endif
 
+  ifeq ($(USE_CUDA),TRUE)
+    ifeq ($(AMREX_CCOMP),gnu)
+      WRAPPER_FLAGS := $(filter-out g++ nvcc, $(shell CC -craype-verbose 2> /dev/null))
+
+      CFLAGS += -Xcompiler='$(WRAPPER_FLAGS)'
+      CXXFLAGS += -Xcompiler='$(WRAPPER_FLAGS)'
+    endif
+  else ifeq ($(USE_MPI),FALSE)
+    ifeq ($(AMREX_CCOMP),gnu)
+      WRAPPER_FLAGS := $(filter-out g++ nvcc, $(shell CC -craype-verbose 2> /dev/null))
```

Comment:
Maybe we should remove the first word instead of explicitly filtering out `g++` and `nvcc`.  something like `$(wordlist 2,1024,$(shell ...))`.  Then we probably don't need to limit this to `gnu`.

Reply:
While I think that's a reasonable adjustment, I don't know if we want to explicitly remove gnu. That might automate this a little too much, i.e. assuming that the flag will always been seen under -craype-verbose is likely not accurate, since the LLVM-based environment isn't a Cray maintained environment.

Until all the PrgEnv's and their wrappers are well defined, I like keeping the gnu check there as a reminder that this was explicitly tailored for that environment for now.

Does that make sense? In either case, which filtering method do you prefer. I can go with either one.

Reply:
We should definitely go with `wordlist` instead of `filter-out g++ nvcc`.  The latter is too hacky.  In fact I am not sure why `nvcc` is there.  For both PrgEnv-nvidia and PrgEnv-nvhpc (at least the default version), CC invokes nvc++, not nvcc.

As for the gnu check, there are currently five programming environments on perlmutter, aocc, cray, gnu, nvhpc and nvidia.  `CC -craype-verbose` works for all.  I don't think we should be concerned about the upcoming LLVM-based environment, whatever that is.  If it's not Cray maintained environment, `CC`, which we explicitly use in other places, is unlikely to work.  So the makefile would need to be fixed anyway whether there is the gnu check or not.

Also, we probably should use `cc` instead of `CC` for `CFLAGS`.  And, in that case, we probably don't need to define `WRAPPER_FLAGS`.

Reply:
LGTM